tests/test_edtfdatestring.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016-2020 CERN.
+#
+# Marshmallow-Utils is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Test the marshmallow EDTF Date String."""
+
+import pytest
+from edtf.parser.grammar import ParseException, level0Expression
+from marshmallow import Schema, ValidationError, fields
+
+from marshmallow_utils.fields import EDTFDateString
+
+
+class MySchema(Schema):
+    f = EDTFDateString()
+
+
+@pytest.mark.parametrize('test_input, expected',
+                         [("2020-01-02garbage", "2020-01-02"),
+                          ("2020-01-45", "2020-01")])
+def test_deserialize_assert(test_input, expected):
+
```

Comment:
Space needed? Maybe for linting but I think not

Reply:
When I remove the spaces on line 22 and 23 and run the tests:
`test_edtfdatestring.py:22:25: E128 continuation line under-indented for visual indent`

Reply:
I meant the break line on 25 but it;s ok

Reply:
Change made